create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -60,6 +60,16 @@
     private static final String SCOPUS_XML_0000469852 = "2-s2.0-0000469852.xml";
     private static final String SCP_ID_IN_0000469852 = "0000469852";
     private static final String DOI_IN_0000469852 = "10.1017/S0960428600000743";
+    private static final String SCOPUS_XML_0018132378 = "2-s2.0-0018132378.xml";
+    private static final String AUTHOR_KEYWORD_NAME_SPACE = "<authorKeywordsTp";
+    private static final String HARDCODED_KEYWORDS_0000469852 = "    <author-keyword xml:lang=\"eng\">\n"
```

Comment:
Can this be placed in resources as a file?

Reply:
is this going to be stored in the end as something like the following?
```
{
...
"keywords": "<author-keyword xml:lang=\"eng\"><sup>64</sup>Cu</author-keyword> "
}?

I think these transformation create a great potential complexity, because  for many fields now we do not know if a field is simple string or an xml string.  It also seems that many fields can be either simple strings or XML, or in the future embedded JSONs, depending on which is the original source. Although I can see that this strategy works for now, it will create many problems in the future.

Reply:
I think what Rurik suggested makes sense, i.e., that XML lines should be stored as base64 strings and in fields that contain the "base64" or something in their name so that we know that these fields need parsing.

Reply:
Save this thought for a later PR :)